### DISCUSSION TITLE
osu freaks out after windows lock

### DISCUSSION THREAD

**Discussion by Devrill** [ppy/osu#20560] [Q&A]
### Type

Game behaviour

### Bug description

Everything is in the video i dont really know how to describe it.
I just windows lock my pc (win+L)
Same thing happens with either borderless or fullscreen

### Screenshots or videos

https://user-images.githubusercontent.com/76632265/193411574-9e49c448-dc96-4ec0-b1e5-e9b8bbd582c2.mp4

### Version

2022.923.1-lazer (latest)

### Logs

[runtime.log](https://github.com/ppy/osu/files/9690401/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9690402/updater.log)
[performance.log](https://github.com/ppy/osu/files/9690403/performance.log)
database and network logs are kinda empty

**Comment by peppy** [comment]
Please try updating your drivers and turning off any overlays you may have running. This isn't normal behaviour and may not be something we can fix from our end.
